RRF-31: search screen

arianaantonio opened the pull request:
- Added searchResultsCollectionView with stub data
- Added product detail screen
- Finetuned search screen

arianaantonio committed:
RRF-31: Search Screen
- Added searchResultsCollectionView with stub data

arianaantonio committed:
RRF-31: Search Screen
- Added product detail screen
- Finetuned search screen

KingOfBrian commented:
There's a bug where the tableview for autocomplete doesn't go away and swiping scrolls the invisible, empty table. If you go in and out of the search field before typing and then try to use it it seems broken.

arianaantonio committed:
RRF-31: Search Screen
- Stopped resetting stub data

arianaantonio committed:
RRF-31: Search Screen
- Fixed highlighting bug
- Show search products view when nothing has been searched
- Disabled scroll on autocomplete table
